deployment/windows/install-jellyfin.ps1

```diff
@@ -82,7 +82,7 @@ if($Quiet.IsPresent -or $Quiet -eq $true){
     }else{
         $Script:InstallServiceAsUser = $true
         $Script:UserCredentials = $ServiceUser
-        $Script:JellyfinDataDir = "C:\Users\$($Script:UserCredentials.UserName)\Appdata\Roaming\jellyfin\"}
+        $Script:JellyfinDataDir = "$env:HOMEDRIVE\Users\$($Script:UserCredentials.UserName)\Appdata\Local\jellyfin\"}
```

Comment:
Can we make this LOCALAPPDATA too? Or is it not defined for service users?

Reply:
LOCALAPPDATA is defined by the context of the running process. If you're in that section of the code you're using service credentials which aren't going to necessarily be those of the user running the install script.